src/standalone/Soil/Biogeochemistry/Biogeochemistry.jl

```diff
@@ -455,6 +456,37 @@ struct SoilCO2FluxBC{F <: Function} <: AbstractSoilCO2BC
     bc::F
 end
 
+"""
+    AtmosFixedCO2FluxBC <: AbstractSoilCO2BC
+
+Set the CO2 concentration to the atmospheric one.
+"""
+struct AtmosFixedCO2FluxBC <: AbstractSoilCO2BC end
```

Comment:
This is replacing a state boundary condition, so I think we might rename this to `AtmosCO2StateBC`?

Reply:
Thanks! I thought I had picked the right one between the two.